language-server/modules/langserver-core/build.gradle

```diff
@@ -148,7 +148,11 @@ test {
     systemProperty "ballerina.home", "$buildDir/"
     systemProperty "experimental", "true"
     systemProperty "ballerina.version", project.version
-    systemProperty "responseTimeThreshold", 2000
+    if (System.getenv('LS_PERFORMANCE_TESTS_IN_GITHUB_WORKFLOW')) {
+        systemProperty "responseTimeThreshold", 4000
```

Comment:
Is the number provided sufficient for a workflow run in GitHub? The value was initially set to `60000`, and the tests still failed. Refer: https://github.com/ballerina-platform/ballerina-lang/blob/73c279c99ad493767f8c73f7345a148d68759b18/language-server/modules/langserver-core/build.gradle#L151

Reply:
Increased the time.